account_payment_term_discount/wizard/account_payment_register.py

```diff
@@ -45,7 +45,7 @@ def onchange_payment_amount(self):
                 payment_date = fields.Date.from_string(self.payment_date)
                 discount_amt = self.invoice_id.discount_amt
 
-                payment_difference = self.payment_difference
+                payment_difference = abs(round(self.payment_difference, 2))
```

Comment:
it would be better to use `float_round` with the `invoice_id.currency_id`'s precision, ultimately the same result for most but it would save headaches for any currency that doesn't use 2 decimal places

Reply:
Hello @hildickethan-S73 I made some changes according to your comment. 

Can you please check again?